doc/tutorial/Tutorial.tex

```diff
@@ -33,23 +33,23 @@ \chapter*{Getting started}
 the \Coq{} Reference Manual or the \textit{Coq'Art}, a book by Y.
 Bertot and P. Castéran on practical uses of the \Coq{} system.
 
-Coq can be used from a standard teletype-like shell window but
-preferably through the graphical user interface
-CoqIde\footnote{Alternative graphical interfaces exist: Proof General
-and Pcoq.}.
+Coq can be tested through a REPL (Read-Eval-Print-Loop) called
+\texttt{coqtop} but for actual developments it is preferable to use
+a graphical user interface such as CoqIDE or
+Proof General\footnote{See \url{https://proofgeneral.github.io/}.}.
 
 Instructions on installation procedures, as well as more comprehensive
 documentation, may be found in the standard distribution of \Coq,
 which may be obtained from \Coq{} web site \url{https://coq.inria.fr/}.
 
-In the following, we assume that \Coq{} is called from a standard
-teletype-like shell window. All examples preceded by the prompting
+The following examples are executed within \texttt{coqtop}.
+All examples preceded by the prompting
 sequence \verb:Coq < : represent user input, terminated by a
 period. 
```

Comment:
I've pushed a commit that masks the reliance on `coqtop`: I suppressed any mention of `coqtop` and of the corresponding commands `Restart.` `Undo.` `Reset Initial.` etc.